fli-iam/shanoir-ng
# Shanoir-issue#77 Delete from PACS - delete dataset one by one w REST

How to test:
- Import some new data
- Check that the data appears in the PACS (http://localhost:8081/dcm4chee-arc/ui2/#/studies)
Delete a newly created dataset:
BEFORE:
- Dataset is still on the local PACS
AFTER
- Dataset is not more on the local PACS

What to test:
- delete datasets one by one
- delete acquisition
- delete examination
- delete full subject

For the laters tests, subject and studies may stay on the PACS, BUT the images should be all deleted.

Closes #77
